added translate to all project

potrohnelox1 opened the pull request:
added translate to all project

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * App-wide internationalization: many UI labels, placeholders and messages in numerous components now use translation keys and locale wiring.
  * Large locale expansions added for English, Armenian, Japanese, Russian and Russian‑Pirate covering onboarding, settings, permissions, device, messaging and feedback texts.

* **Chores**
  * Removed demo/placeholder component.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

potrohnelox1 committed:
Merge branch 'master' into Add-translate

Signed-off-by: potrohnelox1 <221761564+potrohnelox1@users.noreply.github.com>

potrohnelox1 committed:
locale dublicate fix

locale dublicate fix

potrohnelox1 committed:
Fix am tranlate and merge bug

Fix am tranlate and merge bug

ghost commented:
codeRabbit can u work i am not like work

ghost commented:
I have read the CLA Document and I hereby sign the CLA

ghost commented:
I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

I have read the CLA Document and I hereby sign the CLA

ghost commented:
I have read the CLA Document and I hereby sign the CLA

potrohnelox1 committed:
added translate to all project

added translate to all project

potrohnelox1 committed:
Make fix translate
On branch Add-translate
Your branch is up to date with 'origin/Add-translate'.

Changes to be committed:
	modified:   bun.lockb
	modified:   src/components/DevPanel.vue
	modified:   src/components/VideoControl.vue
	modified:   src/components/settings/ApplicationSettings.vue
	modified:   src/components/settings/SoundSettings.vue
	modified:   src/components/settings/spaces/RolesSettings.vue
	modified:   src/locales/am.json
	modified:   src/locales/en.json
	modified:   src/locales/jp.json
	modified:   src/locales/ru.json
	modified:   src/locales/ru_pt.json

potrohnelox1 committed:
Added missing keys

Added missing keys
